mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+# Project information
+site_name: "Uganda EMR Technical Guide"
+site_description: "Technical documentation for Uganda EMR"
+site_author: "METS"
+site_url: "https://github.com/intrahealth/client-registry"
```

Comment:
@AustinBwanika This path is wrong should be https://github.com/METS-Programme/ugandaemr-technicalguide